packages/pyright-internal/src/analyzer/parameterUtils.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * parameterUtils.ts
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ *
+ * Utility functions for parameters.
+ */
+
+import { ParameterCategory } from '../parser/parseNodes';
+import { ClassType, FunctionParameter, TypeCategory } from './types';
+
+export function isTypedKwargs(param: FunctionParameter): boolean {
+    return (
+        param.category === ParameterCategory.VarArgDictionary &&
+        param.type.category === TypeCategory.Class &&
```

Comment:
Use `isClassInstance(param.type)` instead. Otherwise this might be a `type[class]` (an instantiable class).